website/recursion_website/templates/base.html

```diff
@@ -0,0 +1,59 @@
+<!Doctype html>
+<html>
+<head>
+	<title>RECursion</title>

```

Comment:
Title is one of tags important for SEO, can you make the title more verbose? Something like - "RECursion - NIT Durgapur Coding Community"

Reply:
For your reference, SEO is how we optimized our website so that our website is ranked first by a search engine like Google whenever someone searches for keywords like "recursion" or "nit durgapur".
Read point 5 here: https://searchengineland.com/twelve-simple-ways-to-write-search-friendly-html-code-14109
Other important tags are h1, h2, h3, p, alt tag in img.
Every "img" tag should have a "alt" tag containing text with words by which users can find our website on a search engine.